### PR TITLE
Chelakhovl/HW7

### PR DESCRIPTION
**Pull request by Chelakhovl**

### CODE REVIEW

File: Chelakhovl/HW7/task1.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+def return_largestNumber(num1, num2):
```
Comment (kolyasalubov) [inline Chelakhovl/HW7/task1.py:1]: good job

File: Chelakhovl/HW7/task2.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+#NUMBER_PI = 3.14
+
+#def calculates_of_circle (radius):
+#  return  NUMBER_PI * radius ** 2
+#radius = int(input("Enter the radius of the circle: "))
+#result = calculates_of_circle(radius)
+#print(f"The area of the circle is: {result}")
+
```
Comment (kolyasalubov) [inline Chelakhovl/HW7/task2.py:26]: good job

File: Chelakhovl/HW7/task3.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+def calculates_letters(word):
```
Comment (kolyasalubov) [inline Chelakhovl/HW7/task3.py:1]: good job
look at this implementation
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result

**Review by kolyasalubov — APPROVED** [review]

File: Chelakhovl/HW7/task3.py (hunk 3 above)
Comment (Chelakhovl) [reply Chelakhovl/HW7/task3.py:1]: Thank you, I have worked out your option

**Pull request merged**